packages/ember-glimmer/lib/components/link-to.ts

```diff
@@ -72,25 +72,9 @@
   {{/link-to}}
   ```
 
-  any passed value to `disabled` will disable it except `undefined`.
-  to ensure that only `true` disable the `link-to` component you can
-  override the global behavior of `LinkComponent`.
+  any truthy value passed to `disabled` will disable it except `undefined`.
```

Comment:
"except `undefined`" can be removed I think, as `undefined` is not truthy anyways?

Reply:
GAH! Thank you